hawkbit-ui/src/main/resources/messages.properties

```diff
@@ -287,6 +287,7 @@ message.forcequit.action.failed = Force Quitting the action is not possible !
 message.forcequit.action.confirm = Attention!\nForce quit should only be used when the assignment action is not working properly.\nForce quitting an action has no effect on the connected target. It is just resetting \nthe data stored on the SP update server. \nAre you absolutely sure that you want to force quit this action? 
 message.distribution.no.update = distribution {0} set is already assigned to targets and cannot be changed
 message.action.not.allowed = Action not allowed
+message.action.did.not.work = Action did not work. Please try again.
```

Comment:
User has tried to assign a target to the target tag but the target has been deleted. Why should he try again, the target is deleted?